dss/blobstore/__init__.py

```diff
@@ -0,0 +1,152 @@
+import typing
```

Comment:
I believe @ttung has refactored the codebase so that blobstore functionality now lives in the `cloud_blobstore` library. I think this means you need to delete `dss/blobstore/*` and use that library instead. Please work with @ttung to sort this out. You may need to rebase your branch on the latest master.

Reply:
Yes, I discussed this with Tony. I'm moving the new functions to the cloud-blobstore repo

Reply:
done, released new version of cloud-blobstore